RabinKarpAlgorithm/RabinKarp.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RabinKarpAlgorithm
+{
+    public class RabinKarp
+    {
+        int mod = 113;
+        public int GetPatternCount(string baseString, string pattern)
+        {
+            int baseStringLen = baseString.Length;
+            int patternLen = pattern.Length;
+
+            char[] possibleMatchChar = new char[patternLen];
+
+            int patternHash = HashMethod(pattern);
+
+            int cnt = 0;
+            int sum = 0;
```

Comment:
sum cega ?

Reply:
Kontam da i to treba preimenovati, konkretno pamti trenutnu vrednost rolling hash-a i u jednom izrazu izracunava novu. Mislim da bi rollingHashValue bi bio bolji naziv.